install_config/syncing_groups_with_ldap.adoc

```diff
@@ -1206,8 +1212,9 @@ group sync interacts with an LDAP server using the RFC2307 schema.
 |string
 
 |`userNameAttributes`
-|Defines which attributes on an LDAP user entry will be used, in order, as its {product-title} user name. The first attribute with a non-empty value is used. This should match your `PreferredUsername` setting for your `LDAPPasswordIdentityProvider`.
-|string array
+|Defines which attributes on an LDAP user entry will be used, in order, as its {product-title} user name. The first attribute with a non-empty value is used. This should match your `PreferredUsername` setting for your `LDAPPasswordIdentityProvider`. The attribute to use as the name of the user in the {product-title} Group
+record. `mail` or `sAMAccountName` are preferred choices in most installations.
+|string array. Never use a binary attribute as an ID attribute (for example, `objectGUID`). String attributes must be used as the ID, so use `sAMAccountName` or `userPrincipalName` instead.
```

Comment:
Adding the warning to string array does not make sense to me since it applies to all attributes.  And copying that warning to all string and string array fields seems overkill and verbose.  Maybe just include this information as a "for example" in the warning box on the top?  The changes to the description column are fine.